### PR TITLE
update participants requirements + initial configuration file in spanish and english

### PR DESCRIPTION
**Pull request by PatriLoto**
The following commits are in reply to issue number #21 (changes about participants requirements) and   number #31 (introduction-setup)

### CODE REVIEW

**Comment by PatriLoto** [comment]
The last commits refers to issue number 31

File: book/0_Configuracion_inicial.md [hunk 1]
```diff
@@ -1,25 +1,64 @@
...
-## ¿Cómo acceder a los datos disponibles en el sitio Earthdata de la NASA?
+
+![2i2c_login](../assets/2i2c_login.png)
+
+
+
+* Introducí tus credenciales: Ingresa tu nombre de usuario y contraseña (Nota: Para esto deberías haber enviado tu correo asociado a tu cuenta de Github para que se te habilite el acceso con dicha cuenta). 
+
+* Si el acceso es correcto, verás la pantalla que se muestra a continuación. Por último elegí la opción Start para ingresar al ambiente de Jupyter lab en la nube.
+
```
Comment (marielaraj) [inline book/0_Configuracion_inicial.md:17]: elige en vez de elegi

File: book/0_Configuracion_inicial.md [hunk 2]
```diff
@@ -1,25 +1,64 @@
...
 
- Para crear una cuenta en EarthData, seguí este sencillo [tutorial], el cual te guíara paso a paso. Como sugerencia, elige un *nombre de usuario* y *contraseña* que recuerdes bien, ya que los necesitarás más adelante.
-Ahora viene la parte técnica: para acceder a los datos desde programas de Python y Jupyter notebooks, es necesario guardar las credenciales (de EarthData) en un archivo especial.  En este repositorio encontrarás un archivo llamado `.netrc` con un ejemplo (puedes pensar en él como una plantilla). Abre ese archivo y edita la siguiente línea:
+ Para crear una cuenta en EarthData, seguí los pasos que se indica:
+ 
+ * Ingresa al sitio de Earth Nasa: https://www.earthdata.nasa.gov/. Luego selecciona la opción "Use Data" y a continuación "Register". Por último, ingresa a https://urs.earthdata.nasa.gov/.
+
```
Comment (marielaraj) [inline book/0_Configuracion_inicial.md:45]: sigue en vez de segui

File: book/about_the_tutorial/Requirements.md [hunk 3]
```diff
@@ -4,27 +4,33 @@
...
 
-- A free NASA EarthData user account:
+- **A 2i2c hub user account.**
 
-  - To access the data used in this course, you will need a free NASA EarthData user account. You can create an account on the NASA EarthData website ([https://urs.earthdata.nasa.gov/ ](https://urs.earthdata.nasa.gov/)).
+  - To run the course notebooks, you will need a free 2i2c hub account. You can create an account on the 2i2c website (Link).
 
-* A free user account on GitHub:
+* **Fundamentals of Spatial Data.** 
+This course assumes a basic understanding of how geographic information is represented digitally. This includes concepts like points, lines, and areas used to represent locations on Earth.  You also should have knowledge about geospatial concepts such as coordinate systems and data formats.
```
Comment (marielaraj) [inline book/about_the_tutorial/Requirements.md:22]: .. such as coordinate systems and **geographic** data formats.

**Review by marielaraj — COMMENTED** [review]
The comments on the 0_Congiguracion_inicial.md file correspond to use the "tu" instead of "vos".

**Comment by PatriLoto** [comment]
I'll be adding the suggestions soon. Thank you!

File: book/0_Configuracion_inicial.md [hunk 4]
```diff
@@ -1,25 +1,64 @@
...
-* Introduce tus credenciales: Ingresa tu nombre de usuario y contraseña del 2i2c Hub
+  
 
-## ¿Cómo acceder a los datos disponibles en el sitio Earthdata de la NASA?
+
+![2i2c_login](../assets/2i2c_login.png)
+
+
+
+* Introducí tus credenciales: Ingresa tu nombre de usuario y contraseña (Nota: Para esto deberías haber enviado tu correo asociado a tu cuenta de Github para que se te habilite el acceso con dicha cuenta).
```
[the remark this answers is not in the document]
Comment (PatriLoto) [reply book/0_Configuracion_inicial.md:14]: tip added

File: book/about_the_tutorial/Requirements.md (hunk 3 above)
Comment (PatriLoto) [reply book/about_the_tutorial/Requirements.md]: suggestion added

**Comment by PatriLoto** [comment]
File with the steps for the initial configuration (version in english)

**Pull request merged**